build/common/installer/scripts/tomlparser-geneva-config.rb

```diff
@@ -256,27 +274,30 @@ def is_configure_geneva_env_vars()
 file = File.open("geneva_config_env_var", "w")
 
 if !file.nil?
-  file.write("export GENEVA_LOGS_INTEGRATION=#{@geneva_logs_integration}\n")
-  file.write("export GENEVA_LOGS_MULTI_TENANCY=#{@multi_tenancy}\n")
-  if @enable_fbit_threading
-    file.write("export ENABLE_FBIT_THREADING=#{@enable_fbit_threading}\n")
-  end
-
-  if is_configure_geneva_env_vars()
-    file.write("export MONITORING_GCS_ENVIRONMENT=#{@geneva_account_environment}\n")
-    file.write("export MONITORING_GCS_NAMESPACE=#{@geneva_account_namespace}\n")
-    file.write("export MONITORING_GCS_ACCOUNT=#{@geneva_account_name}\n")
-    file.write("export MONITORING_GCS_REGION=#{@geneva_gcs_region}\n")
-    file.write("export MONITORING_CONFIG_VERSION=#{@geneva_logs_config_version}\n")
-    file.write("export MONITORING_GCS_AUTH_ID=#{@geneva_gcs_authid}\n")
-    file.write("export MONITORING_GCS_AUTH_ID_TYPE=AuthMSIToken")
-  end
-  file.write("export GENEVA_LOGS_INFRA_NAMESPACES=#{@infra_namespaces}\n")
-  file.write("export GENEVA_LOGS_TENANT_NAMESPACES=#{@tenant_namespaces}\n")
+  if @disable_linux
+    puts "config::geneva_logs::info:geneva logs integration disabled for linux"
+  else
+    file.write("export GENEVA_LOGS_INTEGRATION=#{@geneva_logs_integration}\n")
```

Comment:
Even if disabled, should we set this as false?

Reply:
> Even if disabled, should we set this as false?


if disabled, none of the GENEVA related env var being set.